src/glfw3.jl

```diff
@@ -228,6 +228,10 @@ const CONTEXT_RELEASE_BEHAVIOR = 0x00022009
 
 const OPENGL_API             = 0x00030001
 const OPENGL_ES_API          = 0x00030002
+const NO_API                 =          0
+
+const TRUE                   =          1
+const FALSE                  =          0
```

Comment:
These two constants appear unused in the Vulkan code. Please don't include them in this PR. If there is a part of the new changes that require bool, I'd like it to work with Julia's `Bool`.